jvb/rootfs/defaults/sip-communicator.properties

```diff
@@ -1,23 +1,4 @@
-org.jitsi.videobridge.SINGLE_PORT_HARVESTER_PORT={{ .Env.JVB_PORT }}
-org.jitsi.videobridge.DISABLE_TCP_HARVESTER={{ .Env.JVB_TCP_HARVESTER_DISABLED }}
-org.jitsi.videobridge.TCP_HARVESTER_PORT={{ .Env.JVB_TCP_PORT }}
+# TODO(brian): not ported in ice4j to new config yet
```

Comment:
Ah I see, so this is a hybrid for a bit?

Reply:
Looks like this is the last property in the file. I've ported it in jitsi/ice4j#215

Reply:
Nice, thank you!

Reply:
I can update this once that gets merged and we update JVB.  Any concern for the case where docker has to handle an older JVB version before this change?